### PR TITLE
Tactic updates to AzureActivity hunting queries

### PR DESCRIPTION
**Pull request by juliango2100**
Fixes #

## Proposed Changes

  - Updated Tactics
  - Updated max time to 14 days
  -

### CODE REVIEW

File: Hunting Queries/AzureActivity/Common_Deployed_Resources.txt [hunk 1]
```diff
@@ -6,7 +6,7 @@
 // in combination with other signals that show suspicious deployment to evaluate if the resource is one
 // that is commonly being deployed/created or unique.
 //
-// DataSource: #AzureActivity
+// DataSource: #AzureActivity #Impact
```
Comment (timbMSFT) [inline Hunting Queries/AzureActivity/Common_Deployed_Resources.txt]: #Impact should be in Tactics line

File: Hunting Queries/AzureActivity/Creating_Anomalous_Number_Of_Resources.txt [hunk 2]
```diff
@@ -7,7 +7,7 @@
 //
 // DataSource: #AzureActivity
 //
-// Tactics: #Execution
+// Tactics: #Execution #Impact
```
Comment (timbMSFT) [inline Hunting Queries/AzureActivity/Creating_Anomalous_Number_Of_Resources.txt]: consistency - should there be a comma here?

File: Hunting Queries/AzureActivity/Common_Deployed_Resources.txt (hunk 1 above)
Comment (juliango2100) [reply Hunting Queries/AzureActivity/Common_Deployed_Resources.txt]: updated

File: Hunting Queries/AzureActivity/Creating_Anomalous_Number_Of_Resources.txt (hunk 2 above)
Comment (juliango2100) [reply Hunting Queries/AzureActivity/Creating_Anomalous_Number_Of_Resources.txt]: updated with comma

**Pull request merged**